src/app/talent/talent/talent.component.ts

```diff
@@ -69,21 +70,20 @@ export class TalentComponent extends AbstractComponent<Talent> {
 
 	private getImageFile(image, name) {
 		return fetch(image)
-		.then(blob => blob.blob())
-		.then((blob) => {
-			
-			return new File([blob], name, {
-				type: 'image/jpeg'
+			.then(blob => blob.blob())
+			.then((blob) => {
+
+				return new File([blob], name, {
+					type: 'image/jpeg'
```

Comment:
you should be able to read the extension of the file from the image data and use it accordingly. think of png files